.github/workflows/promptflow-evals-e2e-test-azure.yml

```diff
@@ -14,21 +14,7 @@ env:
   WORKING_DIRECTORY: ${{ github.workspace }}/src/promptflow-evals
 
 jobs:
-  build:
```

Comment:
Don't we need the same change for other evals pipelines, like regression test and e2e-test-local?

Reply:
Yes, I have removed this code in all other tests, so now they all use editable install.